Source/WebKit/Platform/cocoa/SharedMemoryCocoa.cpp

```diff
@@ -95,17 +96,23 @@ SharedMemory::Handle::~Handle()
     clear();
 }
 
-void SharedMemory::Handle::takeOwnershipOfMemory(MemoryLedger memoryLedger) const
+void SharedMemory::Handle::takeOwnershipOfMemory(const ProcessIdentity& processIdentity, MemoryLedger memoryLedger) const
 {
 #if HAVE(MACH_MEMORY_ENTRY)
     if (!m_port)
         return;
 
+#if HAVE(TASK_IDENTITY_TOKEN)
```

Comment:
This is not going to suffice since support for mach_memory_entry_ownership() taking in an identity token is much more recent than generic identity token support (HAVE(TASK_IDENTITY_TOKEN)).